dev/FHIR3-0-2-MM201901-Cert/Medication-9-0-7/_reference/MPClient/AdministrationAgreement-mp9-59df4530-100a-11e5-b939-0800200c9a66.xml

```diff
@@ -17,7 +17,7 @@
                </tr>
                <tr>
                   <th>Gemaakt</th>
-                  <td>2007-06-20</td>
+                  <td>${DATE, T, D, -144}</td>
```

Comment:
These kind of changes are not in scope for the ticket. Please remove them from the commit.

Reply:
Will not do it. You may consider this as updated fixtures. All improvements are in my view minor and in the range of textual improvements. No significant changes.